src/openmrs-esm-patient-chart.tsx

```diff
@@ -14,3 +14,9 @@ export const bootstrap = lifecycles.bootstrap;
 export const mount = lifecycles.mount;
 export const unmount = lifecycles.unmount;
 export { backendDependencies } from "./openmrs-backend-dependencies";
+export const importTranslation = require.context(
```

Comment:
Since @openmrs/react-root-decorator@3 turns on translations by default, we need to now export an `importTranslations` function from this module.

Reply:
All of the modules with UI strings have to have this boilerplate? Any way to get rid of it?

Reply:
There is no way to get rid of it. Webpack's `require.context()` is the way to load all files within a directory, which we need to do with the translations directory.